sources/odoo.conf

```diff
@@ -4,7 +4,7 @@
 ; --------------------
 ; change this ASAP!!!!
 ;---------------------
-admin_passwd = somesuperstrongadminpasswdYOUNEEDTOCHANGEBEFORERUNNING
+admin_passwd = _ADMIN_PASSWORD_
 
 ; set the default timezone for openerp interface
 timezone = Europe/Paris
```

Comment:
I would go for option 2 indeed. Add a odoo.conf file using a Dockerfile in which the "standard" params are ready to be updated (e.g. `admin_passwd = _ADMIN_PASSWORD_`) so that users can specify their own parameters if necessary